### PR TITLE
Updated minimal metric set

### PR DESCRIPTION
**Pull request by parithosh**

### CODE REVIEW

File: metrics.md [hunk 1]
```diff
@@ -16,37 +16,58 @@ The metrics SHOULD behave under the guidelines set by the [Prometheus documentat
...
-| `beacon_finalized_root`                  | Gauge       | Current finalized root*                                     | On epoch transition  |
-| `beacon_current_justified_epoch`         | Gauge       | Current justified epoch                                     | On epoch transition  |
-| `beacon_current_justified_root`          | Gauge       | Current justified root*                                     | On epoch transition  |
-| `beacon_previous_justified_epoch`        | Gauge       | Current previously justified epoch                          | On epoch transition  |
-| `beacon_previous_justified_root`         | Gauge       | Current previously justified root*                          | On epoch transition  |
+|--------------------------------------------|-------------|-------------------------------------------------------------|----------------------|
+| `libp2p_peer_connected_total`              | Gauge       | Tracks the total number of libp2p peers                     | On peer add/drop     |
+| `beacon_head_state_slot`                   | Gauge       | Slot of the head block of the beacon chain                  | On fork choice       |
```
Comment (rolfyone) [inline metrics.md]: spreadsheet looked like `beacon_head_slot`, and the comment is a little misleading if it's actually in line with the description in the spreadsheet `The latest slot of the beacon chain`
Probably need to make sure that its consistent at least - the label says 'head_state' and the description says 'head block', but overall I think there's probably no reason to deviate from the spreadsheet?

File: metrics.md [hunk 2]
```diff
@@ -16,37 +16,58 @@ The metrics SHOULD behave under the guidelines set by the [Prometheus documentat
...
-| `beacon_finalized_epoch`                 | Gauge       | Current finalized epoch                                     | On epoch transition  |
-| `beacon_finalized_root`                  | Gauge       | Current finalized root*                                     | On epoch transition  |
-| `beacon_current_justified_epoch`         | Gauge       | Current justified epoch                                     | On epoch transition  |
-| `beacon_current_justified_root`          | Gauge       | Current justified root*                                     | On epoch transition  |
-| `beacon_previous_justified_epoch`        | Gauge       | Current previously justified epoch                          | On epoch transition  |
-| `beacon_previous_justified_root`         | Gauge       | Current previously justified root*                          | On epoch transition  |
+|--------------------------------------------|-------------|-------------------------------------------------------------|----------------------|
+| `libp2p_peer_connected_total`              | Gauge       | Tracks the total number of libp2p peers                     | On peer add/drop     |
```
Comment (rolfyone) [inline metrics.md]: I would have thought given that 2 clients said `libp2p_peers` and the old standard was `libp2p_peers`, we could just leave this one as `libp2p_peers`.

File: metrics.md [hunk 3]
```diff
@@ -16,37 +16,58 @@ The metrics SHOULD behave under the guidelines set by the [Prometheus documentat
...
+| `beacon_head_state_slot`                   | Gauge       | Slot of the head block of the beacon chain                  | On fork choice       |
+| `beacon_finalized_epoch`                   | Gauge       | Current finalized epoch                                     | On epoch transition  |
+| `beacon_current_justified_epoch`           | Gauge       | Current justified epoch                                     | On epoch transition  |
+| `beacon_previous_justified_epoch`          | Gauge       | Current previously justified epoch                          | On epoch transition  |
+| `beacon_head_state_active_validators_total`| Gauge       | Current total active validators                             | On epoch transition  |
```
Comment (rolfyone) [inline metrics.md]: should be `beacon_current_active_validators`?

File: metrics.md [hunk 4]
```diff
@@ -16,37 +16,58 @@ The metrics SHOULD behave under the guidelines set by the [Prometheus documentat
...
+| `beacon_fork_choice_reorg_total`           | Counter     | Total number of chain reorganizations                       | On fork choice       |
```
Comment (rolfyone) [inline metrics.md]: any reason this isn't just `beacon_reorgs_total`? all 3 clients have to make changes to go to this new label...

File: metrics.md (hunk 1 above)
Comment (arnetheduck) [reply metrics.md]: is the spreadsheet available somewhere?

Comment (leobago) [reply metrics.md]: Thanks @rolfyone, you are right, we will fix that. 
@arnetheduck I have sent you the spreadsheet by Discord.

Comment (parithosh) [reply metrics.md]: sorry i commited the wrong names by mistake. I fixed it now. The names should reflect the one's we decided in teh spreadsheet.

**Pull request merged**